ecosystem/lava-sdk/src/util/common.ts

```diff
@@ -76,3 +76,28 @@ export function median(values: number[]): number {
     return sortedValues[middleIndex];
   }
 }
+
+export function encodeUtf8(str: string): Uint8Array {
+  return new TextEncoder().encode(str);
+}
+
+export function byteArrayToString(byteArray: Uint8Array): string {
```

Comment:
a duplicate method is in relayer.ts 

https://github.com/lavanet/lava/pull/829/files#diff-a157a78abefeb5ae92d8d4bea24abfa45c3baf2099942e44972ee0cd6721b5e6R261

Reply:
@shleikes small fix

Reply:
Fixed